engine/apps/slack/models/slack_usergroup.py

```diff
@@ -123,59 +124,24 @@ def get_users_from_members_for_organization(self, organization):
         )
 
     @classmethod
-    def update_or_create_slack_usergroup_from_slack(cls, slack_id, slack_team_identity):
-        sc = SlackClientWithErrorHandling(slack_team_identity.bot_access_token)
-        bot_access_token_accepted = True
+    def update_or_create_slack_usergroup_from_slack(cls, slack_id: str, slack_team_identity: SlackTeamIdentity) -> None:
+        sc = SlackClient(slack_team_identity)
+        usergroups = sc.usergroups_list()["usergroups"]
+
+        usergroup = [ug for ug in usergroups if ug["id"] == slack_id][0]
         try:
-            usergroups_list = sc.usergroups_list()
-        except SlackAPIException as e:
-            if e.response["error"] == "not_allowed_token_type":
-                # Trying same request with access token. It is required due to migration to granular permissions
-                # and can be removed after clients reinstall their bots
```

Comment:
> can be removed after clients reinstall their bots

The switch to granular permissions happened a long time ago so I think it's safe to remove this